src/clone_on_capture.rs

```diff
@@ -613,13 +591,13 @@ fn parse_path_expression(data: &mut Data, expr_path: ExprPath) -> Result<Expr> {
             if data.debug {
                 println!("expr_path: segment: {}", path.ident);
             }
-            data.push_usage(Ident::new(
```

Comment:
`Ident::new` panics when token doesn't parse to identifier.

Before this fix this code didn't compile:

```Rust
#[clone_on_capture]
fn test_2() {
    let closure = move || None::<String>;
}
```